formats/json/commonMain/src/kotlinx/serialization/json/Json.kt

```diff
@@ -27,29 +27,48 @@ import kotlin.native.concurrent.*
  * Example of usage:
  * ```
  * @Serializable
- * class DataHolder(val id: Int, val data: String, val extensions: JsonElement)
+ * data class Data(val id: Int, val data: String, val extensions: JsonElement)
  *
- * val json = Json
- * val instance = DataHolder(42, "some data", buildJsonObject { put("additional key", "value") }
+ * val json = Json { prettyPrint = true }
+ * val instance = Data(42, "some data", buildJsonObject { put("key", "value") })
  *
- * // Plain StringFormat usage
- * val stringOutput: String = json.encodeToString(instance)
+ * // Plain Json usage: returns '{"id": 42, "some data", "extensions": {"key": "value" } }'
```

Comment:
Given that you've enabled `prettyPrint = true` for a `json` instance, it will print a different result. I think it is possible here to use a default `Json` or provide another configuration flag in case you need to show instance creation. (But we need to show that we have a default `Json` instance, too).